universal-application-tool-0.0.1/app/services/CfJsonDocumentContext.java

```diff
@@ -0,0 +1,560 @@
+package services;
```

Comment:
most of this code was moved over from `ApplicantData`

Reply:
question definitions, program definitions, and applicant's answers to questions are all stored in JSON